yarn.lock

```diff
@@ -5941,11 +5941,11 @@ lodash.support@~2.3.0:
     lodash._renative "~2.3.0"
 
 lodash.template@^4.4.0:
-  version "4.4.0"
-  resolved "https://registry.yarnpkg.com/lodash.template/-/lodash.template-4.4.0.tgz#e73a0385c8355591746e020b99679c690e68fba0"
-  integrity sha1-5zoDhcg1VZF0bgILmWecaQ5o+6A=
+  version "4.5.0"
+  resolved "https://registry.yarnpkg.com/lodash.template/-/lodash.template-4.5.0.tgz#f976195cf3f347d0d5f52483569fe8031ccce8ab"
+  integrity sha512-84vYFxIkmidUiFxidA/KjjH9pAycqW+h980j7Fuz5qxRtO9pgB7MDFTdys1N7A5mcucRiDyEq4fusljItR1T/A==
   dependencies:
-    lodash._reinterpolate "~3.0.0"
+    lodash._reinterpolate "^3.0.0"
```

Comment:
@lukebooth this change was made when updating to `4.5.0`. What are your thoughts? (cf. [this line](https://github.com/cph/search-with-modifiers/pull/4/commits/f268aa785f83874248c7771f60e404d1a3302b01#diff-8ee2343978836a779dc9f8d6b794c3b2R5731))

Reply:
Looks ok to me. Just locks the major version _and_ only allows patch updates. Probably overkill, but it's not going to hurt anything. 

My only curiosity is why the package is listed twice? (`lodash._reinterpolate@~2.3.0`)

Reply:
I have that question too. They are both listed as dependencies under different packages. If you look through the lock file, you'll notice that it's not the only dependency that is duplicated with different major versions.

Reply:
Ok, if it works it works 😄. Maybe it's something we're supposed to clean up along the way or something. Or maybe this is how yarn is better(?) than bundler where bundler can flog you over and over if all of your gems can't decide on a single version of a dependency 🤷‍♂️